packages/deployer/.gitignore

```diff
@@ -1,2 +1,4 @@
 test/sample-project/deployments
 test/sample-project/contracts/Router.sol
+test/fixture-projects/*/deployments
+test/fixture-projects/*/contracts/Router.sol
```

Comment:
Not sure that we should ignore these:
* deployments - we may wanna keep mainnet deployments around, so perhaps only ignore local ones?
* Router - we may want to keep these around as well. Keep in mind that we have an issue to use deterministic same addressees for all networks, so the Router should be the same no matter the network it targets.

Reply:
Updated it to only ignore `deployments` on `local` & `hardhat`.

But, the `Router` we have to ignore it on the `sample-project` because it includes the branch and commit, and it would be cumbersome for the devs to have to git reset that change every time they do a test on that project and  generate a new Router with a custom branch/commit. Nevertheless, this shouldn't be a problem as the router generation should print always the same.